src/arrays.js

```diff
@@ -1,62 +1,106 @@
 const getNthElement = (index, array) => {
-  // your code here
+  let myIndex = index % array.length;
+  return array[myIndex];
```

Comment:
Because your calling `myIndex` straight away, you don't need to assign it to a variable. It would be cleaner to write:
```
return array[index % array.length];
```